delocate/tools.py

```diff
@@ -148,6 +148,30 @@ def _run(
         ) from exc
 
 
+# Mach-O magic numbers. See:
+# https://github.com/apple-oss-distributions/xnu/blob/5c2921b07a2480ab43ec66f5b9e41cb872bc554f/EXTERNAL_HEADERS/mach-o/loader.h#L65
```

Comment:
Thanks for this - is there anything else you know of that gives Apple's imprimatur to these numbers for some forseeable future?    I see these are official Apple repositories, but it would be reassuring to see some official documentation too.

Reply:
I'm not able to find any detailed Apple documentation for the mach-o format other than the header files. There are a variety of non-Apple blog posts and that describe the format, although none seemed sufficiently thorough to be included here.

Reply:
OK - if that's the best we can do, it's not too bad.